pkg/autoscaler/http_scrape_client_test.go

```diff
@@ -99,6 +106,40 @@ func TestHTTPScrapeClient_Scrape_HappyCase(t *testing.T) {
 	if stat.PodName != "test-revision-1234" {
 		t.Errorf("stat.PodName = %s, want test-revision-1234", stat.PodName)
 	}
+	if stat.ProcessUptime != 0 {
+		t.Errorf("default/missing stat.ProcessUptime = %v, want: 0", stat.ProcessUptime)
+	}
+}
+
+func TestHTTPScrapeClientScrapeHappyCaseiWithOptionals(t *testing.T) {
```

Comment:
nit: (there's an extra `i`)

```suggestion
func TestHTTPScrapeClientScrapeHappyCaseWithOptionals(t *testing.T) {
```